feature-libs/order/components/order-details/order-detail-items/order-consigned-entries/order-consigned-entries.component.html

```diff
@@ -1,7 +1,7 @@
 <div *ngFor="let consignment of consignments" class="cx-list row">
   <ng-template
     [cxOutlet]="OrderOutlets.ORDER_CONSIGNMENT"
-    [cxOutletContext]="{ item: consignment }"
+    [cxOutletContext]="{ item: consignment, order: order }"
```

Comment:
why need pass order data? `consignment` is enough, right?

Reply:
If the consignment has "deliveryPointOfService", then display the deliveryPointOfService data. That's all. No need display `<ng-template let-consignment="consignment" #shippingDetails>`. 
![Screenshot 2023-03-20 at 10 53 34 AM](https://user-images.githubusercontent.com/44440575/226360876-74e39ad1-7d10-41cd-a5c7-0c827c3e3fc8.png)

Reply:
Now, shipping details become duplicate. "Items to be shipped" should not be handled/touched by bopis components.